wordpress-mobile/WordPress-Aztec-iOS
# Copy / paste support for images

Closes #108.

**Description:**

- Implements copy / paste support for images.
- Reduces the visibility for some methods that were `public`.
- Improves the attachment code a bit, and triggers updates only on URL changes.
- Standardizes delegate-type protocols to follow Apple's conventions for naming.

**How to test:**

Test 1:

1. Copy an image from another app (a good source is Pages.app).
2. Paste it in Aztec.

Test 2: Insert an image regularly
Test 3: Insert an image in HTML mode.

For all tests verify the following:

- Make sure the image is properly added to the post.
- Make sure the HTML is updated properly.
- Make sure the layout isn't broken.
